### PR TITLE
The seventh large cleanup of Base 

### PR DESCRIPTION
**Pull request by Yuki2718**
I skipped what will still be blocked by other rules as far as I noticed.
```
! ---------- Obsolete
! /js/ads. /ads.js /pop.js removed; an existing rule .com/ads.$~script,~xmlhttprequest excludes script
! also cleaned up some other exception rules whose counter part bait no more exist
@@/js/ads.js|$script,~third-party,domain=~internethaber.com|~knaqu.tv
@@||uii.io/js/ads.js
@@||sestv.pw/cdn/js/ads.php
@@||dulinks.in/js/ads.js$domain=dulinks.co
@@||img.xcartoon.online/js/ads.js
@@||shrtbtc.com/adblock/js/ads.js$domain=exfaucet.com
@@||ddl.to/js/ads.js
@@||photobucket.com/resources/common/js/ads.js
@@||stayonline.pro/js/ads.js
@@||content-cdn.y2mate.com/themes/js/ads.js
@@||freetutsdownload.net/wp-content/plugins/adblock-monetizer/js/ads.js
@@||cdn.adrise.tv/js/ads.js
@@||gogoanime.cloud/*/js/ads.js
@@||freeiota.de/js/ads.js
@@/themes/v*/js/ads.js$domain=123movies.tf
@@||static.moviescdn.live/themes/v*/js/ads.js
@@/js/ads.js/*$~third-party,script,domain=chillicams.net|waaw.to|openload.casa|meucdn.vip|yandexcdn.com|hqq.to|hindipix.in|waaws.tk|hindipix.in|ventax.net|waaw1.tv|fullbeeg.com|waaw.tv|czxxx.org|newhost.me|videoxseries.xyz|scenelife.org|piratulhd.com|hqq.tv
@@||egyanime.com/layout/js/ads.js
@@||solarmoviefree.ac/themes/v*/js/ads.js
@@||9anime.vip/assets/js/ads.js
@@/js/ads.$script,domain=verifyemailaddress.io
@@/js/ads.js$domain=nutraceuticalsworld.com
@@||web2.0calc.com/js/ads.js
@@||freepsdvn.com/wp-content/themes/mts_sense/js/ads.js
@@||watchfree.su/themes/*/js/ads.js
@@||static.animecdn.xyz/js/ads.js
@@||kisscartoon.is/js/ads.js
@@||short-fly.com/js/ads.js
@@||temp-phone-number.com/files/js/ads.js
@@||simsdom.com/js/ads.*.js
@@||yesmovies.rs/themes/*/js/ads.js
@@||gomovies.ec/themes/*/js/ads.js
@@||123link.vip/js/ads.js
@@||dalatmongm.site/js/ads.js
@@||123link.pw/js/ads.js
@@||mshare.xyz/js/ads.js
@@||mshare.io/js/ads.js
@@||btcmines.club/themes/faucettheme/assets/js/ads.js
@@||kisscartoon.su/js/ads.js
@@||0123movies.is/themes/*/js/ads.js
@@||123link.pro/js/ads.js
@@||solarmoviez.su/themes/*/js/ads.js
@@||gomovies.lc/themes/*/js/ads.js
@@||watchfree.at/themes/*/js/ads.js
@@||static.xmovies88.online/themes/*/js/ads.js
@@||wakeupcallme.com/js/ads.js
@@||prppsn.com/templates/base_master/js/ads.js
@@||gostream.nu/themes/v*/js/ads.js$~third-party
@@||videodistribution.com/js/ads.js
@@||g7t7z7m6.stackpathcdn.com/js/ads.js
@@||hungama.ind.in/assets/js/ads.js$domain=hungama.com
@@||kissasian.es/js/ads.js$~third-party
@@||temp-mails.com/files/js/ads.js
@@||cartoon8.tv/js/ads.js
@@||kimcartoon.co/js/ads.js
@@||watchcartoonsonline.me/js/ads.js
@@||giveaway-club.com/includes/js/ads.js
@@||anime-loads.org/assets/pub/js/ads.com/ads.js
@@||static.animecdn.stream/js/ads.js
@@||wallpaperstudio10.com/assets/js/ads.min.js
@@||9ani.me/js/ads.js
@@||updato.b-cdn.net/wp-content/themes/updato/js/ads.js$domain=updato.com
@@||kisscartoon.ac/js/ads.js^
@@||assets.galaxant.com/js/ads.js
@@||kissanime.ac/js/ads.js
@@||kimcartoon.io/js/ads.js
@@||kissanime.co/js/ads.js
@@||3rabshort.com/js/ads.js
@@||coinb.ink/js/ads.js
@@||download.themeslide.com/js/ads.js
@@||egy-links.com/js/ads.js
@@||cut-earn.com/js/ads.js
@@||globaldjmix.com/dist/js/ads.js
@@||common-static.tunnl.com/player/js/ads.js
@@||video-static.tunnl.com/js/ads.js
@@||bit-url.com/js/ads.js
@@||static.xmovies88.stream/themes/*/js/ads.js
@@||ecut.io/js/ads.js
@@||ecut.it/js/ads.js
@@||adlinkme.com/js/ads.js
@@||igram.im/js/ads.js
@@||cut-egy.ml/js/ads.js
@@||cut-egy.ga/js/ads.js
@@||kisscartoon.es/js/ads.js
@@||shorteurl.com/js/ads.js
@@||cuturlink.com/js/ads.js
@@||url.gem-flash.com/js/ads.js
@@||lolskinlistgenerator.com/Kodlar/JS/ads.js
@@||link-cash.com/js/ads.js
@@||adshorte.com/js/ads.js
@@||adshort.co/js/ads.js
@@||shortfly.us/js/ads.js
@@||i-invdn-com.akamaized.net/js/ads.js
@@||adshort.me/js/ads.js
@@||coinlink.us/js/ads.js
@@||adbilty.me/js/ads.js
@@||windowstechinfo.com/wp-content/plugins/detect-adblock/js/ads.js
@@||1movies.*/js/ads.js^
@@||mydaddy.cc/js/ads.js
@@||hackinformer.com/wp-content/plugins/detect-adblock/js/ads.js
@@||cutwin.com/js/ads.js
@@/js/ads.js$~third-party,domain=clik.pw|linksh.top
@@||igeeksblog.com/wp-content/plugins/wp-quads-pro/assets/js/ads.js
@@||asmwall.com/js/ads.js|$~third-party
@@||transparentcalifornia.com/static/js/ads.js
@@||swedroid.se/js/ads.js^
@@||online-converting.com/js/ads.js^
@@||l2s.io/js/ads.js
@@||haxmaps.com/js/ads.js
@@||tune.pk/js/ads.js^
@@||cinesift.com/js/ads.js
@@||adbull.me/js/ads.js
@@||urle.co/js/ads.js^
@@||xmovies8.ru/themes/*/js/ads.js
@@||kisscartoon.io/js/ads.js^
@@||common-static.vooxe.com/player/js/ads.js
@@||pinoybay.ch/assets/js/ads.js
@@||vidoza.net/js/ads.js
@@/wp-content/themes/mts_ad_sense/js/ads.js^$~third-party,domain=320box.com|adisteme.kz|adobevn.net|aggressivecars.com|allbluevr.com|androidcrush.com|applosungen.com|appluseis.com|awamiweb.com|azfinans.ru|besthealthyguide.com|bhookedcrochet.com|biochemden.com|bloggingden.com|braininspired.net|business-solution.us|carajuki.com|carapraktis.info|caratulasparacuadernos.com|catchmyjobs.com|ccna6.com|ccnav6.com|cineview.me|circuiteasy.com|conference-hosting.net|conspatriot.com|dailyhealthkeeper.com|darmawan.my.id|dayoadetiloye.com|degree-classes.com|detutorial.com|diedit.com|diysite.tips|diyzone.tips|dollsnews.com|dormir-moins-bete.com|download-album.com|duniaq.com|espiaresgratis.com|everythingsohealth.com|filmreno.net|freesmshub.com|gamingguidetips.com|geekpeaksoftware.com|gennarino.org|gratiserotik.se|grfeed.com|healthspiritbody.com|healthysolutionsbenefit.com|healthytotal.net|healthzone.tips|hightechbuzz.net|horamerica.com|howtorootmobile.com|idownloadblogs.com|inmigrantesenmadrid.com|insurancequotestip.com|kampusexcel.com|kimdirbu.com|lahuria.com|laivanduc.com|langsungviral.com|learn-automation.com|librospdfgratis.net|lifeblogid.com|liriklaguindonesia.net|lotsofhealthy.com|martketingdigital.com|masterdrivers.com|minimilitiacheats.com|mjemagazine.com|mobilehardreset.com|mobilesreview.co.in|mstory.me|mypapercraft.net|mysemakan.com|newmoviesfun.com|nicecarsinfo.com|ofertasahora.com|onlinehealthsociety.com|orduh.com|organichealthcorner.com|otimundo.com|ovnihoje.com|parhaye.com|pointsrecipes.com|prdnews.com|quoteacademy.com|ricasaude.com|rwaag.org|sapsibubapa.org|sarungpreneur.com|satgobmx.com|shoemoney.com|smartphonehrvatska.com|soltvperu.com|ssante.com|sumberpendidikan.com|supercarcenter.com|supportcanon.com|tatuajestatu.com|techcrack.net|teclane.com|tellynews.us|thestreetriders.com|tipsdroidmax.com|todaysjobalert.com|todolandia.com|top10remedioscaseros.com|topnaturaltips.com|topwebsitelists.com|trenddify.com|truquesedicas.org|tutorialescomo.com|ultra4kporn.com|usefuldiys.com|vegrecetas.com|vidabytes.com|viewhealth.net|viralholic.net|viralistics.com|wikigain.com|wisdomquotesandstories.com|www-mabadi.com|yoemigro.com|yomasti.co|zenfoneasus.com
@@||liveedu.tv/static/js/ads.js
@@||bitcoinaliens.com/*/js/ads.js
@@||angrybirdsnest.com/wp-content/themes/abn-responsive/js/ads.js
@@||5.79.78.12/js/ads.js
@@||policelifes.de/js/ads.js
@@||mid-day.com/Resources/middaymobile/js/ads.js
@@||indiatimes.com/*/wp-content/*/js/ads.js
@@||wallpapermania.eu/assets/js/ads.js
@@||static.filehorse.com/js/ads.js
@@||porntube.com/*/js/ads.js
@@||cdn.vipboxsa.co/js/ads.js
@@||cdn.vipbox.tv/js/ads.js


@@||webcafe.bg/js/ads.js
@@||sledujufilmy.cz/js/ads.js
@@||sledujserialy.sk/theme/js/ads.js
@@||vibilagare.se/sites/all/themes/custom/okweb/js/ads.js
@@||ebimg.dk/misc/js/ads.js
@@||static.lexpress.fr/js/ads.js
@@||port.hu/js/ads.min.js
@@||itascan.info/js/ads.block.min.js
@@||rockitecn.nohup.it/web/js/ads.js
@@||itascan.info/js/ads.js
@@||acquavivalive.it/TemplateLive/app/assets/js/ads.js
@@||ani19.net/js/ads.js
@@||delfi.ee/misc/crs/js/ads.js|$domain=delfi.ee
@@||satkurier.pl/frontEnd/js/ads.js
@@||politiken.dk/static/content/js/ads.js$domain=ekstrabladet.dk
@@||static.hdonline.vn/template/frontend/js/ads.js
@@||linkneverdie.com/Assets/Js/ads.js


@@||global.fncstatic.com/static/v/all/js/ads.js
@@||publico.es/publico/js/ads.min.js
@@||video.sanook.com/assets/vi/js/Ads.js



@@||playtube.ws/js/pop.js
@@||onlystream.tv/js/pop.js
@@||statics.mystream.to/pop.js
@@||cdn.vup.to/js/pop.js
@@||flixsite.b-cdn.net/js/pop.js
@@||rapidstream.co/js/pop.js
@@||gounlimited.to/js/pop.js|
@@||titsbox.com/js/pop.js
@@||clipwatching.com/js/pop.js
@@||thevideobee.to/js/pop.js
@@||zedstream.com/js/pop.js
@@||xvidstage.com/js/pop.js
@@||zmovs.com/js/pop.js
@@||vivud.com/js/pop.js
@@/pop.js|$domain=fistfast.net
@@||vidoza.net/js/pop.js
@@||fistfast.com/pop.js
@@||dato.porn/js/pop.js|
@@||arasa.news/pop.js
@@||watchingmysistergoblack.com/pop.js
/pop.js|$domain=pornve.com
@@||multiup.org/pop.js
@@||163.172.29.124/pop.js
@@||upvast.com/pop.js

! /advertisement.$~script,domain=~advertisement.solutions.zalando.com
@@||adgaterewards.com/assets/js/vc/advertisement.js|$~third-party
@@||codelist.cc/advertisement.js
@@/advertisement.js|$domain=fistfast.net

@@||wcoanimedub.tv/inc/embed/ads.js
@@||linkvertise.com/assets/ads.js
@@||popsplit.us/dist/ads.js
@@||ninja.io/js/dist/ads.js
@@||seadragons.io/ads.js
@@||chompers.io/ads.js
@@||thewatchcartoononline.tv*inc/embed/ads.js
@@||robofight.io/*/ads.js
@@||minigiants.io/ads.js
@@||wcoforever.com/inc/embed/ads.js
@@||brutalmania.io/ads.js
@@||vanis.io/ads.js
@@||camarchive.tv/assets/ads.js
@@||ievaphone.com/public/javascripts/ads.js
@@||call2friends.com/*/javascripts/ads.js
@@||cdn.reignn.com/Products/Reignn/DataPixels/AdBlockerCheck/ads.js
@@||restplay.xyz/player/assets/ads.js
@@||notebookchat.com/ads.js
@@||soup.io/ads.js
@@||freebcc.org/claim/vendor/ads/ads.js
@@||viptools.es/assets/ads.js
@@||c.yandexcdn.com/js/adv/ads.js$domain=yandexcdn.com
@@||kxcdn.com/js/wutime_adblock/ads.js$domain=enxf.net
@@||asianrun.com/ads.js
@@||books-share.com/scripts/ads.js
@@||promods.net/ads.js
@@||megaup.net/ads.js
@@||tagmp3.net/ads.js
@@||adshorted.net/ads.js
@@||sombex.com/ads.js
@@||vegashipcalc.co.uk/scripts/ads.js
@@||cruisin.me/community/js/wutime_adblock/ads.js
@@||v7player.wostreaming.net/ads.js
@@||mycastapp.*.amazonaws.com/assets/ads.js$domain=mycast.io
@@||bc.vc/fly/ads.js
@@||tinyhomebuilders.com/scripts/ads.js
@@||dreamdth.com/*/ads.js
@@||satellite-calculations.com/ads.js
@@||ratemyteachers.com/ads.js
@@||rmdown.com/ads.js
@@||aosmark.com/ads.js
@@||couponscorpion.com/ads.js
@@||dramavery.com/ads.js
@@||game-maps.com/ads.js
@@||tubidy.net/ads.js
@@||extremereportbot.com/ads.js
@@||asianen.com/ads.js
@@||bspin.club/themes/demo/assets/javascript/ads.js
@@||lunaticfiles.com/ads.js|
@@||reuters.com/ads.js|
@@||mp4upload.com/ads.js
@@||tetris.com/play-tetris-content/ads.js
@@||a2zp30.com/ads.js
@@||evowars.io/ads.js
@@||itsourtree.com/ads.js
@@||a2zapk.com/ads.js
@@||thop.in/ads.js
@@||viewasian.tv/ads.js
@@||assets.beinsports.com/*/scripts/ads.js$~third-party
@@||houstonpress.com/js_cache/ads.js
@@||phoenixnewtimes.com/js_cache/ads.js^
@@||benkhouya.com/ads.js$domain=anonymousemail.me
@@||cartoonhd.pw/templates/cartoonhd/assets/scripts/ads.js
@@||expertblogz.com/*/ads.js
@@||cdncontent.wakanim.tv/scripts/dist/ads.js
@@||pasty.link/javascript/ads.js
@@||speedtest.net/javascript/ads.js
@@||img.tfd.com/ads.js
@@||sltrib.com/pb/resources/scripts/utils/ads/ads.js
@@||clix4btc.com/ads.js
@@||hotcopper.com.au/ads.js
@@||stream.nbcolympics.com/assets/page/general/ads.js
@@||abysstream.com/ads.js
@@||viewdrama.com/ads.js
@@||redditgirls.com/ads.js
@@||arconaitv.us/assets/javascripts/ads.js
@@||estrenos10.com/assets/ads.js
@@/ads.js$domain=cut-urls.com,~third-party
@@||hdliveextra-a.akamaihd.net/HD/scripts/*/config/ads.js
@@||arconai.tv/assets/javascripts/ads.js
@@||xmovies8.es/themes/*/ads.js?v=
@@||wallpapersite.com/scripts/ads.js
@@||assets.gameallianz.com/api/js/ads/ads.js
@@||player.gogoanime.ch/static/ads.js^
@@||player.9cartoon.me/static/ads.js^
@@||tubeninja.net/ads.js^
@@||uploadboy.com/ads.js
@@||d.ibtimes.co.uk/imasdk/ads.js^
@@||gelbooru.com/ads.js
@@||onlinemschool.com/script/ads.js
@@||nbc.com/generetic/scripts/ads.js
@@||gamenguide.com/common/js/common/ads.js
@@||itechpost.com/common/js/common/ads.js
@@||eenadu.net/ads.js
@@||retainguaninefluorite.info/path/ads/ads.js
@@||uploadrocket.net/ads.js
@@||wallpapershome.com/scripts/ads.js
@@||receivesmsonline.net/ads.js


@@||okanime.tv/ads.js
@@||okanime.com/ads.js
@@||masrawy.com/Scripts/ads.js^
@@||encoretvb.com/ads.js
@@||chocotv.com.tw/scripts/ads.js
@@||warforum.cz/ads.js
@@||version2.dk/ads.js
@@||patrika.com/scripts/ads.js
@@||jagran.com/*/ads.js
@@||bhaskar.com/*/ads/ads.js|
@@||ynet.co.il/common/javascript/ads.js
@@||everyeye.it/adv/ads.js
@@||dailybest.it/ads.js
@@||calciomercato.com/ads.js
@@||js.iplsc.com/inpl.abd/latest/ads.js?$domain=pomponik.pl|interia.pl|js.iplsc.com
@@||realmadryt.pl/ads.js
@@||static.webcamera.pl/webcamera/ads.js
@@||s-pt.ppstatic.pl/o/js/pp-ads/ads.js
@@||gamekit.io/ads.js
@@||pieseauto.ro/functions/javascript/ads.js
@@||moduri.ro/wp-content/themes/parabola/ads.js
@@||topsinhalamp3.com/scripts/ads.js
@@||web-static.linetv.tw/public/scripts/ads.js
@@||fullcrackpc.com/ads.js
@@||nhacpro.net/ads.js
@@/ads.js$domain=hayhaytv.vn|talktv.vn|chiasenhac.vn|hplus.com.vn|fptplay.net|phim.media

! /advert.$~script,domain=~advert.ae|~advert.club|~advert.ee|~advert.io|~advert.media|~advert.org.pl|~motortrader.com.my
@@||radiosantaclaus.com/js/advert.js
@@/advert.js$domain=radiostreamlive.com,~third-party
```

### CODE REVIEW

**Comment by Yuki2718** [comment]
With this change, most of old script rules like `#%#window.canRunAds = true;` can probably be removed.

**Comment by Alex-302** [comment]
> With this change, most of old script rules like `#%#window.canRunAds = true;` can probably be removed.

That exclusions does not cover all cases. Only exlusion+scriptlet can be removed. But I am not sure. Third-party filters can add blocking rules.

**Comment by Yuki2718** [comment]
> .Only exlusion+scriptlet can be removed. 

That's what I meant, as I saw many instances such that:
```
@@||example.com/ads.js
example.com#%#window.canRunAds = true;
```

> But I am not sure. Third-party filters can add blocking rules.

AFAIK, CJX's EasyList Lite and Tofu (豆腐) filter still includes these baits, but both are not in the stock list of AG. We can't take care of all the lists on the Internet, better to focus only on the stock lists - we already removoed `#@##adsense` filters as such.

**Comment by Alex-302** [comment]
Also Estonian filter has `/js/ads.`

**Comment by Yuki2718** [comment]
> Also Estonian filter has `/js/ads.`

Confirmed, do you prefer me to revert `/js/ads.` rule? Another option will be simply badfiltering the rule.

**Comment by Alex-302** [comment]
> do you prefer me to revert

No. We can exclude this rule in FiltersRegistry. Will check later.

**Comment by wout-codes** [comment]
What's the point of cleaning up? It's risky since the link might work in some other countries.

**Comment by Yuki2718** [comment]
@wout-codes Um, what? You want to keep thousands of obsolete filters which have no benefit and only consume memory - if not battery too? EasyList has been cleaning them up in large scale for more than a year and uAssets regularly cleans them up, this is, frankly, where AG is relatively behind of them and I'm trying to fill this gap in. Or you think that AG provides Optimized filters is just a marketing gimmick? What I cleaned up here are obsolete whitelists where the blocking counterparts were removed from EasyList - I skipped what will still be blocked by other rules if I removed whitelist. Only Estonian list still includes a blocking rule which should simply be removed, these are common bait for adblock detection.

**Comment by Yuki2718** [comment]
In this case particular, what I removed are whitelists - there's a report in paper that obsolete whitelist were exploited to deliver ads.

**Comment by wout-codes** [comment]
> @wout-codes Um, what? You want to keep thousands of obsolete filters which have no benefit and only consume memory - if not battery too? EasyList has been cleaning them up in large scale for more than a year and uAssets regularly cleans them up, this is, frankly, where AG is relatively behind of them and I'm trying to fill this gap in. Or you think that AG provides Optimized filters is just a marketing gimmick? What I cleaned up here are obsolete whitelists where the blocking counterparts were removed from EasyList - I skipped what will still be blocked by other rules if I removed whitelist. Only Estonian list still includes a blocking rule which should simply be removed, these are common bait for adblock detection.

I see. Worth it then.

**Comment by Yuki2718** [comment]
Removed ad servers added to EL but different in third-party:
```
! third-party difference
||flippantreprehensiblecharming.com^$third-party
||odiumdrawlip.com^$third-party
||wtg-ads.com^
```

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -7025,7 +6986,6 @@ dobryruch.co.uk#$#.adsbygoogle { position: absolute!important; left: -3000px!imp
 ! https://github.com/AdguardTeam/AdguardFilters/issues/53284
 ! https://github.com/AdguardTeam/AdguardFilters/issues/50437
 @@||js.iplsc.com/inpl.abd/*/proxy.html?host=$generichide
-@@||js.iplsc.com/inpl.abd/latest/ads.js?$domain=pomponik.pl|interia.pl|js.iplsc.com
```
Comment (Alex-302) [inline EnglishFilter/sections/foreign.txt:7028]: Cannot be removed. Matched by many rules.

**Comment by Alex-302** [comment]
Checked adservers.txt, foreign.txt, whitelist.txt
antiadblock.txt will be next

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (Yuki2718) [reply EnglishFilter/sections/foreign.txt:7028]: Yeah you're right, the path include lots of baits.

**Comment by Alex-302** [comment]
I mean I removed from master the rules from that files. You can just remove changes in that files from PR.

**Comment by Alex-302** [comment]
And please don't merge master into this PR. I will remove unnecessary rules by parts.

**Comment by Yuki2718** [comment]
@Alex-302 All right, then it'll be better to remove from master and don't merge this PR. I don't know how to exclude individual file in PR other than reverting the changes.

**Comment by Alex-302** [comment]
Just leave as is) It is not critical issue, will be processed later.

**Comment by Yuki2718** [comment]
@Alex-302 This can also be removed: `@@||games.softgames.com/assets/advertisement.js`

**Pull request closed without merging**